<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataChooser</class>
 <widget class="QMainWindow" name="DataChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Berlin Sans FB</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>310</width>
      <height>130</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>userID1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>userID2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Compare them</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>90</y>
      <width>501</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>trajectory graph </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Top10 POI</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>380</y>
      <width>281</width>
      <height>148</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>userID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>Analyze it</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
